src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -11588,57 +11589,69 @@ def test_aks_azure_service_mesh_with_egress_gateway(
             "aks mesh enable-egress-gateway --resource-group={resource_group} --name={name} "
             "--egress-gateway-nodeselector istio=egress"
```

Comment:
@biefy wondering if we should also remove the --nodeselector param from these tests, just to be safe. Or should we do that in a follow-up?

Reply:
In retrospect, let's finalize the design first and then confirm what to do about nodeSelector first.